Corona-Warn-App/src/main/res/values-de/antigen_strings.xml

```diff
@@ -168,9 +168,9 @@
     <!--RAT profile onboarding-->
     <string name="rat_profile_onboarding_image_content_description">Eine Frau mit einem Smartphone in der Hand steht vor einem Gebäude. Ein QR-Code symbolisiert das zu scannende Schnelltest-Profil.</string>
     <!-- XTXT: Create RAT profile onboarding title -->
-    <string name="rat_profile_onboarding_title">Legen Sie Ihre persönlichen Daten als QR-Code ab, um die Registrierung an der Teststelle zu beschleunigen.</string>
+    <string name="rat_profile_onboarding_title">Legen Sie Ihre persönlichen Daten als QR-Code ab, das Sie per QR-Code bei jeder teilnehmenden Teststelle vorlegen können.</string>
```

Comment:
```suggestion
    <string name="rat_profile_onboarding_title">Legen Sie Ihre persönlichen Daten als QR-Code ab, die Sie per QR-Code bei jeder teilnehmenden Teststelle vorlegen können.</string>
```

Reply:
I would rather suggest "Legen Sie Ihre persönlichen Daten als QR-Code ab, den Sie bei jeder teilnehmenden Teststelle vorlegen können."
To avoid duplication of "QR-Code".
@CV113

Reply:
Ok, updated suggestion ☝🏼

Reply:
Alright, @CV113, @KathrinLuecke let me know which one I should go with.